pkgs/applications/misc/bemoji/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bemoji";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "marty-oehme";
+    repo = pname;
+    rev = version;
+    hash = "sha256-XXNrUaS06UHF3cVfIfWjGF1sdPE709W2tFhfwTitzNs=";
+  };
+
+  strictDeps = true;
+  dontBuild = true;
+
+  installPhase = ''
+    chmod +x bemoji
+    install -D bemoji $out/bin/bemoji
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/marty-oehme/bemoji/";
+    license = licenses.mit;
+    description = "Emoji picker with support for bemenu/wofi/rofi/dmenu and wayland/X11";
+    maintainers = with maintainers; [ laurent-f1z1 ];
```

```suggestion
    mainProgram = "bemoji";
    maintainers = with maintainers; [ laurent-f1z1 ];
```

`meta.mainProgram` should be set in the case of a package with some main binary (see #246386)